pkg/apis/extensions/types.go

```diff
@@ -413,6 +421,10 @@ const (
 	// Replace the old daemons by new ones using rolling update i.e replace them on each node one after the other.
 	RollingUpdateDaemonSetStrategyType DaemonSetUpdateStrategyType = "RollingUpdate"
 
+	// Replace the old daemons by new ones using rolling update i.e replace them on each node one
+	// after the other, creating the new pod and then killing the old one.
```

Comment:
this strategy wouldn't work well for pods that make use of resources like host ports, right? (since the new pod would not be able to bind to the port while the old one was still running, and therefore wouldn't ever become healthy, assuming its health check actually indicated whether it was successfully bound and running). that doesn't mean the strategy isn't useful, but that limitation should probably be called out somewhere.